### ISSUE TITLE
Download instructions are insecure: use of http URL's

### ISSUE THREAD

**Issue by vinc17fr** [speed47/spectre-meltdown-checker#155]
The new download instructions are:
```
curl -L meltdown.ovh -o spectre-meltdown-checker.sh
wget meltdown.ovh -O spectre-meltdown-checker.sh
```
but both curl and wget will connect to meltdown.ovh on port 80 (http), which is not secure. A https URL should be provided.

**Comment by speed47** [comment]
Heheh, fair enough. done!

**Comment by vinc17fr** [comment]
Not that you should also update the instructions at the beginning of the spectre-meltdown-checker.sh script too.

**Comment by speed47** [comment]
fixed in 053f161
